reference/orders.v3.yml

```diff
@@ -2909,22 +2786,78 @@ components:
           description: |
             The type of resource with which the metafield is associated.
           enum:
-            - category
             - order
             - brand
             - product
             - variant
+            - category
+          example: order
         resource_id:
-          type: string
+          type: integer
           description: |
             The unique identifier for the resource with which the metafield is associated.
-          maxLength: 10000000000
+          example: 0
+          readOnly: true
+      required:
+        - permission_set
+    MetafieldBase_Post:
```

Comment:
Created a model for a metafield base that doesn't include `resource_id`. At least for orders, the `resource_id` seems like it is always going to be taken from the `order_id` path param to show the relationship:

![image](https://user-images.githubusercontent.com/29524703/174182141-aa83a81e-2260-4aa4-96a9-c305f5654ed5.png)